app/frontend/src/service/client.ts

```diff
@@ -93,7 +93,7 @@ if (process.env.NODE_ENV === "development") {
     apis.pages,
     apis.references,
     apis.requests,
-    apis.search,    
```

Comment:
I ran `yarn format` and it adjusted this tab here. Let me know if I should recommit without this change.

Reply:
Only a small change here so it's cool to keep this in this case :)